packages/wallet/contract/src/utils.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+
+import { assert, details as X, quote as q } from '@agoric/assert';
+import { deeplyFulfilledObject, objectMap } from '@agoric/internal';
+import { E } from '@endo/far';
+
+/**
+ *
+ * @param {AmountKeywordRecord} amountKeywordRecord
+ * @param {(brand: Brand) => ERef<Purse>} purseForBrand
```

Comment:
Nit: Rename to `getPurseForBrand` (here and below). I thought that this was just a single purse reference based on the name.

Reply:
The function does just look up a single purse for a single brand.